src/schoology-extractor/README.md

```diff
@@ -40,7 +40,7 @@ poetry run coverage report
 
 ## Output
 
-Four files in the `./output` directory:
+Four files in the project(or the specified output) directory:
```

Comment:
In Jira I mentioned making sure the default directory is documented. This does document it, so that request is satisfied.